### PR TITLE
Updated readme with Felis Cloud

### PR DESCRIPTION
**Pull request by lprost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,29 +17,35 @@ You can install the provider using `pip`:
 pip install qiskit-alice-bob-provider
 ```
 
-`pip` will handle installing all the python dependencies automatically and you
-will always install the  latest (and well-tested) version.
+`pip` will handle installing all the python dependencies automatically and
```
Comment (MaxleDrut) [inline README.md:20]: "will install the latest release with all its dependencies automatically"

File: README.md [hunk 2]
```diff
@@ -17,29 +17,35 @@ You can install the provider using `pip`:
...
-## Remote execution on Alice & Bob QPUs: use your API key
+The provider is currently **NOT** compatible with Qiskit 1.0. We recommend
+creating a new environment before installing it.
 
-To obtain an API key, please [contact Alice & Bob](https://alice-bob.com/contact/).
+## Remote execution on Alice & Bob QPUs with a Felis Cloud subscription
 
-You can initialize the Alice & Bob remote provider using your API key
-locally with:
+When subscribing to Felis Cloud on the
```
Comment (MaxleDrut) [inline README.md:28]: "Subscribing to Felis Cloud allows you to generate API Keys" (not just to get one)

File: README.md (hunk 1 above)
Comment (lprost) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (lprost) [reply README.md:28]: I would not fix this: most users only need one key, and the text doesn't suggest you can't generate more keys.

Comment (MaxleDrut) [reply README.md:28]: Fine for me then

**Review by MaxleDrut — APPROVED** [review]
Remember to squash your commits!

**Pull request closed without merging**